modules/ROOT/pages/salesforce/salesforce-connector-xml-maven.adoc

```diff
@@ -0,0 +1,112 @@
+= Salesforce - XML and Maven Support
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+XML and POM file configuration information, and XML-related usage topics.
+
+[[xmlnamespace]]
+== XML Namespace
+
+If you are creating Salesforce connector XML, add the namespace for the connector. By contrast, Studio creates the namespace for a connector automatically when you create a project for a Mule app.
```

Comment:
What is the customer actually doing? Creating or configuring a Salesforce connector using XML?

Reply:
Where is the namespace defined?